inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/queue/pulsar/PulsarOperator.java

```diff
@@ -292,6 +292,9 @@ public boolean topicExists(PulsarAdmin pulsarAdmin, String tenant, String namesp
             }
             for (String t : topicList) {
                 t = t.substring(t.lastIndexOf("/") + 1); // not contains /
+                if (!isPartitioned) {
```

Comment:
Exception cound be thrown if the topic(non-partitioned topic) is not suffixed with `-partition-`